src/pages/settings/Wallet/WalletPage/WalletPage.tsx

```diff
@@ -64,29 +82,28 @@ function WalletPage({bankAccountList, cardList, fundList, isLoadingPaymentMethod
     });
     const [showConfirmDeleteModal, setShowConfirmDeleteModal] = useState(false);
 
+    // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
     const hasBankAccount = !_.isEmpty(bankAccountList) || !_.isEmpty(fundList);
     const hasWallet = !_.isEmpty(userWallet);
-    const hasActivatedWallet = _.contains([CONST.WALLET.TIER_NAME.GOLD, CONST.WALLET.TIER_NAME.PLATINUM], userWallet.tierName);
+    const hasActivatedWallet = ([CONST.WALLET.TIER_NAME.GOLD, CONST.WALLET.TIER_NAME.PLATINUM] as string[]).includes(userWallet?.tierName ?? '');
     const hasAssignedCard = !_.isEmpty(cardList);
```

Comment:
```suggestion
    const hasAssignedCard = !isEmptyObject(cardList);
```

Reply:
Filip refers to `import {isEmptyObject} from '@src/types/utils/EmptyObject';`